integration-tests/test_stdin_to_stdout.py

```diff
@@ -1,6 +1,8 @@
 import json
 import os
 
+import pytest
+
 from common.utils import run_job
```

Comment:
you should install the isort extension, so the imports will be sorted the same:
```
import json
import os

import pytest
from common.utils import run_job
```

Reply:
have some conflict with this plugin. aren't they sorted now?

Reply:
no, and `pytest` is unused. I'm fixing.